Engines/Wine/Verbs/vcrun2003/script.js

```diff
@@ -20,14 +20,10 @@ Wine.prototype.vcrun2003 = function () {
     this.wizard().wait(tr("Please wait while {0} is installed...", "Microsoft Visual C++ 2003 Redistributable (x86)"));
     this.run(setupFile, "/S", null, false, true);
 
-    var dlls = [
-        "msvcp71",
-        "mfc71"
-    ];
+    const dlls = ["msvcp71", "mfc71"];
 
-    var that = this;
-    dlls.forEach(function (dll) {
-        cp(that.programFiles() + "/BZEdit1.6.5/" + dll, this.system32directory());
+    dlls.forEach(dll => {
```

Comment:
Is it possible to make Codacy find the places where `function()` is used instead of `=>`?

Reply:
I think https://eslint.org/docs/rules/prefer-arrow-callback does what you want

Reply:
Do you want to try it?